app/templates/first/tiny_image_template.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  <h1>Tiny Image Directive</h1>
+  <img src="{{url}}" height= "100px" width="100px" alt="{{title}}">
+</div
```

Comment:
Be aware that you are passing properties all the way down to the template that you are not using.